src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -589,9 +589,24 @@ private void DynamoView_Loaded(object sender, EventArgs e)
             };
             BackgroundPreview.SetBinding(VisibilityProperty, vizBinding);
             TrackStartupAnalytics();
-        }
 
-        private void TrackStartupAnalytics()
+            // In native host scenario (e.g. Revit), the "Application.Current" will be "null". Therefore, the InCanvasSearchControl.OnRequestShowInCanvasSearch
+            // will not work. Instead, we have to check if the Owner Window (DynamoView) is deactivated or not.  
+            if (Application.Current == null)
+                {
+                this.Deactivated += (s, args) => { HidePopupWhenWindowDeactivated(); };
+                }
+        }
```

Comment:
Can the above indentation be fixed? Maybe caused by a mixture of spaces and tabs.